src/fidesops/service/connectors/mongodb_connector.py

```diff
@@ -45,7 +45,10 @@ def client(self) -> MongoClient:
         """Returns a client for a MongoDB instance"""
         config = MongoDBSchema(**self.configuration.secrets or {})
         uri = config.url if config.url else self.build_uri()
-        return MongoClient(uri, serverSelectionTimeoutMS=5000)
+        try:
+            return MongoClient(uri, serverSelectionTimeoutMS=5000)
+        except ValueError:
+            raise ConnectionException("Value Error connecting to MongoDB.")
```

Comment:
Are we suppressing the error message here because it could contain the secrets?

Reply:
yes, they often do contain secrets, but this is probably going too far in the other direction and suppresses all useful information.  A good future improvement would provide more information without displaying secrets.

Reply:
ah - I should also clarify that this was put here to begin with because a ValueError causes a 500 error to be thrown